tests/mypy/mypy-default.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+follow_imports = silent
+strict_optional = True
+warn_redundant_casts = True
+warn_unused_ignores = True
+disallow_any_generics = True
+check_untyped_defs = True
+
+# for strict mypy: (this is the tricky one :-))
+disallow_untyped_defs = True
```

Comment:
I think currently the only way to specify plugins in mypy is via config file. So in order to test with *and* without the plugin (or with different plugin settings), we would need to have different config files to point mypy at during `mypy.api.run`. So I started by copying the relevant bits from `setup.cfg`.